Cargo.toml

```diff
@@ -13,7 +13,7 @@ default = ["pyo3/num-bigint", "pyo3/auto-initialize"]
 
 [dependencies]
 pyo3 = { version = "0.16.5" }
-cairo-rs = { git = "https://github.com/lambdaclass/cairo-rs.git", rev = "c22a11db4e22bf324534c7b6f978a40478f85d50" }
+cairo-rs = { git = "https://github.com/lambdaclass/cairo-rs.git", rev = "1908502521c615b2bf2c87b0f7afbc879e861826" }
```

Comment:
Todo
Update commit once https://github.com/lambdaclass/cairo-rs/pull/643 is merged